nitro-protocol/test/contracts/MultiAssetHolder/transfer.test.ts

```diff
@@ -36,31 +36,33 @@ const addresses = {
-    ${' 2. overfunded      -> 1 EOA'}      | ${{c: 2}}  | ${{A: 1}}             | ${[0]}       | ${{A: 0}}             | ${{c: 1}}       | ${{A: 1}}       | ${undefined}
-    ${' 3. underfunded     -> 1 EOA'}      | ${{c: 1}}  | ${{A: 2}}             | ${[0]}       | ${{A: 1}}             | ${{}}           | ${{A: 1}}       | ${undefined}
-    ${' 4. funded      -> 1 channel'}      | ${{c: 1}}  | ${{C: 1}}             | ${[0]}       | ${{C: 0}}             | ${{c: 0, C: 1}} | ${{}}           | ${undefined}
-    ${' 5. overfunded  -> 1 channel'}      | ${{c: 2}}  | ${{C: 1}}             | ${[0]}       | ${{C: 0}}             | ${{c: 1, C: 1}} | ${{}}           | ${undefined}
-    ${' 6. underfunded -> 1 channel'}      | ${{c: 1}}  | ${{C: 2}}             | ${[0]}       | ${{C: 1}}             | ${{c: 0, C: 1}} | ${{}}           | ${undefined}
-    ${' 7. -> 2 EOA         1 index'}      | ${{c: 2}}  | ${{A: 1, B: 1}}       | ${[0]}       | ${{A: 0, B: 1}}       | ${{c: 1}}       | ${{A: 1}}       | ${undefined}
-    ${' 8. -> 2 EOA         1 index'}      | ${{c: 1}}  | ${{A: 1, B: 1}}       | ${[0]}       | ${{A: 0, B: 1}}       | ${{c: 0}}       | ${{A: 1}}       | ${undefined}
-    ${' 9. -> 2 EOA         partial'}      | ${{c: 3}}  | ${{A: 2, B: 2}}       | ${[1]}       | ${{A: 2, B: 1}}       | ${{c: 2}}       | ${{B: 1}}       | ${undefined}
-    ${'10. -> 2 chan             no'}      | ${{c: 1}}  | ${{C: 1, X: 1}}       | ${[1]}       | ${{C: 1, X: 1}}       | ${{c: 1}}       | ${{}}           | ${undefined}
-    ${'11. -> 2 chan           full'}      | ${{c: 1}}  | ${{C: 1, X: 1}}       | ${[0]}       | ${{C: 0, X: 1}}       | ${{c: 0, C: 1}} | ${{}}           | ${undefined}
-    ${'12. -> 2 chan        partial'}      | ${{c: 3}}  | ${{C: 2, X: 2}}       | ${[1]}       | ${{C: 2, X: 1}}       | ${{c: 2, X: 1}} | ${{}}           | ${undefined}
-    ${'13. -> 2 indices'}                  | ${{c: 3}}  | ${{C: 2, X: 2}}       | ${[0, 1]}    | ${{C: 0, X: 1}}       | ${{c: 0, X: 1}} | ${{C: 2}}       | ${undefined}
-    ${'14. -> 3 indices'}                  | ${{c: 5}}  | ${{A: 1, C: 2, X: 2}} | ${[0, 1, 2]} | ${{A: 0, C: 0, X: 0}} | ${{c: 0, X: 2}} | ${{A: 1, C: 2}} | ${undefined}
-    ${'15. -> reverse order (see 13)'}     | ${{c: 3}}  | ${{C: 2, X: 2}}       | ${[1, 0]}    | ${{C: 2, X: 1}}       | ${{c: 2, X: 1}} | ${{}}           | ${reason1}
-    ${'16. incorrect fingerprint        '} | ${{c: 1}}  | ${{}}                 | ${[0]}       | ${{}}                 | ${{}}           | ${{A: 1}}       | ${reason2}
+    name                                   | heldBefore | isSimple | setOutcome            | indices      | newOutcome            | heldAfter       | payouts         | reason
+    ${' 0. channel not finalized        '} | ${{c: 1}}  | ${true}  | ${{}}                 | ${[0]}       | ${{}}                 | ${{}}           | ${{A: 1}}       | ${reason0}
```

Comment:
it is difficult to parse this diff. An additional column `isSimple` has been added. Existing rows have this field set to `true`. An additional row has been added, which is the same as case 1 but with `isSimple: false`.